src/main/java/techreborn/blockentity/storage/item/StorageUnitBaseBlockEntity.java

```diff
@@ -54,11 +58,15 @@
 public class StorageUnitBaseBlockEntity extends MachineBaseBlockEntity
 	implements InventoryProvider, IToolDrop, IListInfoProvider, IContainerProvider {
 
-
 	// Inventory constants
 	private static final int INPUT_SLOT = 0;
 	private static final int OUTPUT_SLOT = 1;
 
+	// NBT keys
+	private static final String TAG_LOCKED_ITEM = "lockedItem";
+
+	private static final Item NULL_ITEM = Items.AIR;
```

Comment:
I am unsure if this isn't causing a race with the static initializer of `Items.AIR`.

Reply:
I would use an itemstack here, as it should lock on the NBT value as well.